cli/provider/service.go

```diff
@@ -83,14 +83,17 @@ func (n *ServiceProvider) GetService(ctx context.Context, cmd string) (interface
 	case "config", "update":
 		return tools.NewTools(n.logger, tel), nil
 	// TODO: add case for mock
-	case "record", "test", "mock":
+	case "record", "test", "mock", "normalize":
 		commonServices := n.GetCommonServices(*n.cfg)
 		if cmd == "record" {
 			return record.New(n.logger, commonServices.YamlTestDB, commonServices.YamlMockDb, tel, commonServices.Instrumentation, *n.cfg), nil
 		}
 		if cmd == "test" {
 			return replay.NewReplayer(n.logger, commonServices.YamlTestDB, commonServices.YamlMockDb, commonServices.YamlReportDb, tel, commonServices.Instrumentation, *n.cfg), nil
 		}
+		if cmd == "normalize" {
```

Comment:
Is there any other interface we are planning to add in the future that leads to this duplicity?

Reply:
No, merged the if conditions.